astropy/visualization/tests/test_units.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import io
+
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    HAS_PLT = False
+else:
+    HAS_PLT = True
```

Comment:
Oh, okay.  I didn't realize it was that old--just surprised we weren't using this already since I know it's been a requested feature in the past :D